committee-steering/governance/faq.md

```diff
@@ -0,0 +1,100 @@
+# FAQ
+
+## Why is it so long
+
+These documents attempt to ensure the institutional knowledge
+of individuals managing SIGs today is well documented and available
+to all members in the SIG.  There are a surprising number of responsibilities and
+processes that have developed as part of facilitating management
+of the project.
+
+## Why so much structure
+
+These documents intentionally leans toward more structure rather than
+less.  There are 2 reasons for this:
+
+- It is easy for SIGs to remove or edit sections in this
+  template to reduce structure, whereas it is harder to make up new
+  structure from scratch
+- The SIG governance questionnaire showed a consistent preference for
+  more structure over less.  (almost all answers were to have
+  the same or more amount of structure)
+
+## Why use mailing lists for tracking roles
+
+Using mailing lists to track roles makes it simple to contact
+the members of a SIG with a given role.  This is necessary for
+escalation and decision making to occur.  Alternatives considered
+were: OWNERs files, OWNERs alias', and GitHub teams.  However,
+none of these have a simple way to send a message to the members.
+
+SIGs may choose to have OWNERs and GitHub teams that mirror the
+mailing list membership.
+
+## Why track membership as a special role
+
+Decision making in SIGs is [lazy consensus][lazy-consensus] driven,
+however it is important to define the individuals participating in
+lazy consensus. This should be the individuals responsible for the
+continued health and future direction of the area.  Individuals that
+do not actively make contributions to the area may be consulted, but
+are not ultimately responsible for decision making.
+
+## How do the results from the survey fit in
+
+The feedback from the SIG Charter survey are largely consistent
+with the decisions in this document.  It is worth noting the
+>50% of the results came from only 2 SIGs, so the results are
+skewed to the experiences in those SIGs.
+
+### Question: What to focus on
+
+- 35% - Discussion and resolution of decisions
```

Comment:
I don't we should reference %'s as surveys change over time.

Reply:
Do you mean peoples opinions or the survey results?  The survey is closed and not accepting additional responses.  I will double check the %'s.